content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifcategoryfield/columnitempreview/columnitempreview.html

```diff
@@ -18,9 +18,9 @@
         <coral-columnView-preview-label>${'Title' @ i18n}</coral-columnView-preview-label>
         <coral-columnview-preview-value>${helper.title}</coral-columnview-preview-value>
 
-        <sly data-sly-test.ccifId="${helper.properties.ccifId}">
+        <sly data-sly-test.cifId="${helper.properties.cifId}">
```

Comment:
Shouldn't we have an equivalent change to rename that property somewhere?

Reply:
Curiously enough we had already cifId in the properties and it was not displayed, so this fixes a hidden bug as well.

Reply:
We should then probably add a test for that, I think we only have one for the category properties page (https://github.com/adobe/commerce-cif-connector/blob/master/it/http/src/test/java/com/adobe/cq/commerce/it/http/GraphqlProductConsoleIT.java#L225)

Reply:
(we can add that test later, didn't mean it for this PR)